rest/tests/test_rest.py

```diff
@@ -535,8 +568,32 @@ def test_immutable_fields(self):
             "vocabulary": -1,
         }
         result = self.create_term(
-            repo_dict['slug'], vocab_dict['slug'], term_dict,
-            skip_assert=True
+            self.repo.slug, vocab_slug, term_dict, skip_assert=True
+        )
+        # these keys should be different since they are immutable or set by
+        # the serializer
+        self.assertNotEqual(term_dict['id'], result['id'])
```

Comment:
It's important in making sure we don't miss a field and let the user edit fields they shouldn't be allowed to